internal/hmsds/hmsds-tx-postgres.go

```diff
@@ -668,7 +668,8 @@ func (t *hmsdbPgTx) InsertComponentTx(c *base.Component) (int64, error) {
 		enabledFlg = *c.Enabled
 	}
 	// Normalize key
-	normID := xnametypes.NormalizeHMSCompID(c.ID)
+	var normID string
+	normID = xnametypes.NormalizeHMSCompID(c.ID)
```

Comment:
Why was this change made?

Reply:
Opened up https://github.com/OpenCHAMI/smd/pull/67